core/templates/dev/head/pages/exploration-editor-page/feedback-tab/feedback-tab.directive.ts

```diff
@@ -72,9 +72,10 @@ angular.module('oppia').directive('feedbackTab', [
           };
           // Fetches the threads again if any thread is updated.
           ctrl.fetchUpdatedThreads = function() {
```

Comment:
Maybe we should call this fetchUpdatedThreadsAsync?

Reply:
The code base seems to use the `fetch` prefix to imply async already, so I think that'd be overkill.

Reply:
OK, seems like that's a reasonable convention too.